x-pack/platform/plugins/shared/observability_ai_assistant/server/routes/runtime_types.ts

```diff
@@ -91,15 +80,10 @@ export const conversationCreateRt: t.Type<ConversationCreateRequest> = t.interse
 export const conversationUpdateRt: t.Type<ConversationUpdateRequest> = t.intersection([
   baseConversationRt,
   t.type({
-    conversation: t.intersection([
-      t.type({
-        id: t.string,
-        title: t.string,
-      }),
-      t.partial({
-        token_count: tokenCountRt,
-      }),
-    ]),
+    conversation: t.type({
```

Comment:
won't this break updates of existing conversations?

Reply:
Good point. We'd need to filter out the token count first. Probably easier to just leave this as optional.